paasta_tools/automatic_rollbacks/state_machine.py

```diff
@@ -114,10 +114,10 @@ def start_timer(self, timeout, trigger, message_verb):
 
         timer_start = time.time()
         timer_end = timer_start + timeout
-        formatted_time = datetime.datetime.fromtimestamp(timer_end)
+        formatted_time = datetime.datetime.fromtimestamp(timer_end).strftime("%X")
 
         self.notify_users(
-            f"Will {message_verb} in {timeout} seconds, (at {formatted_time})"
+            f"Will {message_verb} in {timeout} seconds, (at {formatted_time})! Press 'Disable auto rollbacks' to cancel this."
```

Comment:
`start_timer` is used for more than just the automatic rollback countdown -- it's also used for the auto-abandon countdown and the auto-complete countdown. Also, the text `Disable auto rollbacks` is defined by MarkForDeploymentProcess.

Probably best to add a parameter to `start_timer` called `cancel_button_text` or something, and use that. And since afaik we don't currently allow you to cancel an auto-certify or auto-abandon countdown, you'll either need to add buttons for those or omit this part of the message `if cancel_button_text is None`.